src/systems/thruster/Thruster.cc

```diff
@@ -171,6 +174,13 @@ void Thruster::Configure(
     this->dataPtr->fluidDensity = _sdf->Get<double>("fluid_density");
   }
 
+  // Get a custom topic.
+  if (_sdf->HasElement("topic"))
+  {
+    this->dataPtr->topic = ignition::transport::TopicUtils::AsValidTopic(
```

Comment:
We should be avoiding repeating the `ignition::` namespace so it's easier to port to Garden

```suggestion
    this->dataPtr->topic = transport::TopicUtils::AsValidTopic(
```

Reply:
Removed in [b551759](https://github.com/gazebosim/gz-sim/pull/1681/commits/b5517593ef7453f569c5c0e23f203bb266fee6f1).